docs/src/examples/components/Popup/Usage/PopupExampleCloseButton.shorthand.tsx

```diff
@@ -0,0 +1,96 @@
+import { useBooleanKnob } from '@stardust-ui/docs-components'
+import * as React from 'react'
+import { Button, Dropdown, Popup, Flex } from '@stardust-ui/react'
+
+const inputItems = [
```

Comment:
we want to make examples as short as possible; can we make this just a minimal:

```typescript
const inputItems = ['Bruce Wayne', 'Natasha Romanoff' ]
```

I guess it's enough for this example. Would be great to open a PR and go through `Dropdown` examples and clean them up a little bit so that users can focus on what's relevant to that particular example. 

fyi @layershifter